apps/src/gamelab/spritelab/Spritelab.js

```diff
@@ -0,0 +1,33 @@
+var commands = require('./commands');
+
+var Spritelab = function() {
+  window.p5.prototype.executeDrawLoopAndCallbacks = function() {
+    this.drawSprites();
+  };
+
+  window.p5.prototype.setBackground = function(color) {
```

Comment:
Each block will have a corresponding function here, that basically just calls `commands.function` (which is where the real logic of the block lives).

Reasoning for this- blocks must correspond to uniquely named functions, and this allows us to consolidate the logic where possible. putting the wrapper functions on the p5 object allows the interpreter to call out to them, but then the bulk of the code is still isolated.